mlir/test/Dialect/Affine/parallelize.mlir

```diff
@@ -0,0 +1,133 @@
+// RUN: mlir-opt %s -allow-unregistered-dialect -affine-parallelize| FileCheck %s
+
+// for multiple nested for-loops 
+#map0 = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
+#map1 = affine_map<() -> (0)>
+#map2 = affine_map<() -> (64)>
+#map3 = affine_map<() -> (8)>
+#map4 = affine_map<() -> (1)>
+#map5 = affine_map<(d0, d1, d2, d3, d4, d5, d6, d7) -> (d0 + d1, d2 * 2 + d3, d4 * 2 + d5, d6 + d7)>
+#map6 = affine_map<() -> (3)>
+// CHECK-DAG: [[MAP0:#map[0-9]+]] = affine_map<(d0, d1, d2, d3) -> (d0, d1, d2, d3)>
+// CHECK-DAG: [[MAP1:#map[0-9]+]] = affine_map<() -> (0)>
+// CHECK-DAG: [[MAP2:#map[0-9]+]] = affine_map<() -> (64)>
+// CHECK-DAG: [[MAP3:#map[0-9]+]] = affine_map<() -> (8)>
+// CHECK-DAG: [[MAP4:#map[0-9]+]] = affine_map<() -> (1)>
```

Comment:
No need to match constant maps. The constants will appear inline and you won't see these maps used typically.